### PR TITLE
add ctype family of functions

### PR DESCRIPTION
**Pull request by drdzyk**
This PR adds [ctype](https://www.php.net/manual/en/book.ctype.php) family of functions.

### CODE REVIEW

File: builtin-functions/_functions.txt [hunk 1]
```diff
@@ -709,6 +709,20 @@ function rtrim ($s ::: string, $what ::: string = " \n\r\t\v\0") ::: string;
 function xor_strings ($s ::: string, $t ::: string) ::: string;
 function similar_text ($first ::: string, $second ::: string, float &$percent = TODO) ::: int;
 
+function extension_loaded(string $extension): bool;
```
Comment (drdzyk) [inline builtin-functions/_functions.txt:712]: This function needed for launching tests from zend repo. Namely each test for ctype_* function in zend repo has next code:
```
<?php	if (!extension_loaded('ctype')) die('skip ctype extension not available');?>
```

Comment (Ramzeeset) [reply builtin-functions/_functions.txt:712]: Let's write declaration in common style like functions above
```suggestion
function extension_loaded($extension ::: string) ::: bool;
```

File: builtin-functions/_functions.txt [hunk 2]
```diff
@@ -709,6 +709,20 @@ function rtrim ($s ::: string, $what ::: string = " \n\r\t\v\0") ::: string;
...
+
+function ctype_alnum(mixed $text): bool;
```
Comment (Ramzeeset) [inline builtin-functions/_functions.txt:714]: Same for these functions

Comment (drdzyk) [reply builtin-functions/_functions.txt:714]: why?

**Review by Ramzeeset — APPROVED** [review]

**Pull request merged**